nucliadb_relations/src/errors.rs

```diff
@@ -31,7 +30,7 @@ pub enum RelationsErr {
     #[error("IO error: {0}")]
     IOError(#[from] std::io::Error),
     #[error("Disk error: {0}")]
-    DiskError(#[from] DiskErr),
+    DiskError(#[from] FsError),
```

Comment:
Regarding my previous comment, those two variants can be combined IMO

Reply:
Same response as with the `FsError` suggestion.